redis/src/main/scala/zio/redis/TestExecutor.scala

```diff
@@ -27,6 +32,22 @@ private[redis] final class TestExecutor private (
     res: => RespValue.BulkString
   ): USTM[BulkString] = STM.succeedNow(if (input.isEmpty) errResponse(command) else res)
 
+  private val SAddCommand        = api.Sets.SAdd[String, String]
+  private val SCardCommand       = api.Sets.SCard[String]
+  private val SDiffCommand       = api.Sets.SDiff[String, String]
+  private val SDiffStoreCommand  = api.Sets.SDiffStore[String, String]
+  private val SInterCommand      = api.Sets.SInter[String, String]
+  private val SInterStoreCommand = api.Sets.SInterStore[String, String]
+  private val SIsMemberCommand   = api.Sets.SIsMember[String, String]
+  private val SMembersCommand    = api.Sets.SMembers[String, String]
+  private val SMoveCommand       = api.Sets.SMove[String, String, String]
+  private val SPopCommand        = api.Sets.SPop[String, String]
+  private val SRandMemberCommand = api.Sets.SRandMember[String, String]
+  private val SRemCommand        = api.Sets.SRem[String, String]
+  private val SUnionCommand      = api.Sets.SUnion[String, String]
+  private val SUnionStoreCommand = api.Sets.SUnionStore[String, String]
+  private val SScanCommand       = api.Sets.SScan[String, String]
+
```

Comment:
I'd leave this one as it was before, just to reduce the diff, as we'll have to address the test executor thoroughly anyhow. If that's not possible, please ignore this comment.

Reply:
Yeah, that's brings compilation errors(
But I pulled out command names from RedisCommand into constants, and has returned almost like was before, without names. What do u think about that?